varia/sass/blocks/cover/_style.scss

```diff
@@ -2,9 +2,12 @@
 .wp-block-cover-image {
 
 	background-color: #{map-deep-get($config-cover, "color", "background")};
-	min-height: #{map-deep-get($config-cover, "height")};
 	margin: inherit;
 
+	&:not([style*="min-height"]) {
```

Comment:
Is this necessary? If there is an inline style setting a minimum height the theme CSS will be overridden anyway?

Reply:
Yeah, I think this one is necessary so that Cover Blocks from before 6.4 fallback to the default cover height variable in child themes. 

Older Cover Blocks are not automatically converted to use the new inline `min-height` (I dont think) so we need something to make sure that the height remains unchanged without it. Once a customer converts the cover block to 6.4 or updates the height setting in the editor, than this selector gets ignored entirely.

Reply:
You're right about the min-height won't automatically be added to cover blocks on the front-end. It will only be added when you change the height from the default in the editor. If that's what we both seeing, I believe we don't need it :) Because where we currently have the min-height will be the default on the front-end for both old and new cover blocks. And when you change the height the inline style takes over.

Just to be clear, what I'm suggesting doesn't make anything better :) My point is the following two CSS will be exactly the same, and the changes made in this file doesn't change anything.

```
.wp-block-cover,
.wp-block-cover-image {
	min-height: #{map-deep-get($config-cover, "height")}; 
}
```

```
.wp-block-cover:not([style*="min-height"]),
.wp-block-cover-image:not([style*="min-height"]) {
	min-height: #{map-deep-get($config-cover, "height")}; 
}
```